javacutil/src/main/java/org/checkerframework/javacutil/trees/NewTreeUtils.java

```diff
@@ -0,0 +1,149 @@
+package org.checkerframework.javacutil.trees;
+
+import com.sun.source.tree.ExpressionTree;
+import com.sun.source.tree.Tree;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.List;
+import javax.lang.model.SourceVersion;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.javacutil.BugInCF;
+
+/**
+ * This class contains util methods for reflective accessing Tree classes and methods that were
+ * added after Java 8.
+ */
+public class NewTreeUtils {
```

Comment:
I can't think of a better name.  Instead of a new class, I could move the contents of this class into `TreeUtils`.

Reply:
How about `TreeUtilsAfterJava8`?